.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/blob/v0.212.0/containers/javascript-node/.devcontainer/base.Dockerfile
+# [Choice] Node.js version (use -bullseye variants on local arm64/Apple Silicon): 16, 14, 12, 16-bullseye, 14-bullseye, 12-bullseye, 16-buster, 14-buster, 12-buster
+ARG VARIANT="16-bullseye"
```

Comment:
We should stick to the latest node version supported by AWS Lambda (14 for now unfortunately). We need to have clearer guidelines on this ... Discussion in progress https://github.com/awslabs/aws-lambda-powertools-typescript/discussions/331

Reply:
sure. i can keep it at node 14 level